debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/MySqlConnectorConfig.java

```diff
@@ -782,6 +782,23 @@ public static SecureConnectionMode parse(String value, String defaultValue) {
                     + "'never' to specify the connector should never run a snapshot and that upon first startup the connector should read from the beginning of the binlog. "
                     + "The 'never' mode should be used with care, and only when the binlog is known to contain all history.");
 
+    public static final Field SNAPSHOT_ISOLATION_MODE = Field.create("snapshot.isolation.mode")
```

Comment:
This definition and related methods could also be extracted into `RelationalDatabaseConnectorConfig`.
See `RelationalDatabaseConnectorConfig.TABLE_IGNORE_BUILTIN` and `MySqlConnectorConfig.TABLES_IGNORE_BUILTIN` to see how a config option can be added the the common class and overridden below if needed.
Also the new config options should be added to ConfigDef.

Reply:
does the Field group position ordering matter? or it just needs to be unique?

I can see that we have a check on the uniqueness of the position in JsonSchemaCreatorService.

Basically, I was thinking of moving this field to the new position to avoid position conflicts across connectors.

Reply:
I've moved this field to the parent and changed its group position to a unique number.